### PR TITLE
Added my profile

### PR DESCRIPTION
**Pull request by CraigP17**

### CODE REVIEW

**Comment by CraigP17** [comment]
Hacktoberfest tag

File: index.html [hunk 1]
```diff
@@ -1184,7 +1184,34 @@ <h4 class="mb-0">
       </div>
     </div>
   </div>
+
```
Comment (ShehrozIrfan) [inline index.html:1187]: You can remove this extra line.

File: index.html [hunk 2]
```diff
@@ -1184,7 +1184,34 @@ <h4 class="mb-0">
...
+            </a>
+            <a href="https://www.linkedin.com/in/CPinto17" target="_blank">
+              <i class="fa fa-linkedin-square fa-2x icon-link icon-link-linkedin" aria-hidden="true"></i>
+            </a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+
```
Comment (ShehrozIrfan) [inline index.html:1214]: You can remove this extra line.

**Comment by ShehrozIrfan** [comment]
Hey @CraigP17 !
Great work. But the PR's for this repository are excluded for Hacktoberfest.
You can visit the [Ruby Rocks](https://github.com/ShehrozIrfan/ruby-rocks) for contribution to Hacktoberfest, where you can write any code in ruby programming language and it will be accepted.

Have fun 🙌🏼

**Pull request merged**